tests/common.py

```diff
@@ -120,6 +120,12 @@ def __init__(self, *args, **kwargs):
 
         self.check = None
 
+    def reset(self):
+        self.check = None
```

Comment:
@yannmh added `force_reload` in `run_check` could that be enough? Let's try to keep this class neat.